Runtime/Scripts/MeshSyncServer.Properties.cs

```diff
@@ -0,0 +1,306 @@
+                        sendChanges = true;
+                    }
+                }
+
+                // Send updated entities:
+                {
+                    foreach (var kvp in GetClientObjects())
+                    {
+                        var entity = kvp.Value;
+
+                        if (entity.trans == null)
+                        {
+                            continue;
+                        }
+
+                        if (!entityTransforms.TryGetValue(entity, out var matrix))
+                        {
+                            entityTransforms.Add(entity, entity.trans.localToWorldMatrix);
+                        }
+                        else
+                        {
+                            // Check if object moved:
+                            if (entity.trans.localToWorldMatrix != matrix)
+                            {
+                                SendTransform(kvp.Key, entity);
+
+                                entityTransforms[entity] = entity.trans.localToWorldMatrix;
+                            }
+                        }
+
+#if AT_USE_SPLINES
+                        if (changedSplines.Count > 0 && entity.dataType == EntityType.Curve)
+                        {
+                            //m_server.SendCurve(entity, kvp.Key);
+                            SendCurve(kvp.Key, entity);
+                            sendChanges = true;
+                        }
+#endif
+#if AT_USE_PROBUILDER && UNITY_EDITOR
+                        if (entity.dataType == EntityType.Mesh && entity.proBuilderMeshFilter != null)
+                        {
+                            if (changedMeshes.Contains(entity.proBuilderMeshFilter))
+                            {
+                                SendMesh(kvp.Key, entity);
+                                sendChanges = true;
+                            }
+                        }
+#endif
+                    }
+
+#if AT_USE_SPLINES
+                    changedSplines.Clear();
+#endif
+
+#if AT_USE_PROBUILDER && UNITY_EDITOR
+                    changedMeshes.Clear();
+#endif
+                }
+
+                if (needsClientSync)
+                {
+                    needsClientSync = false;
+                    sendChanges = true;
+
+                    m_server.RequestClientSync();
+                }
+
+                if (sendChanges)
+                {
+                    CurrentPropertiesState = PropertiesState.Sending;
+                    onSceneUpdateEnd -= SceneUpdated;
+                    onSceneUpdateEnd += SceneUpdated;
+                    m_server.MarkServerInitiatedResponseReady();
+                }
+            }
+        }
+
+        void SendTransform(string path, EntityRecord entity)
+        {
+            // TODO: Implement this. https://jira.unity3d.com/browse/BLENDER-478
+        }
+
+
+#if AT_USE_PROBUILDER && UNITY_EDITOR
+        public void MeshChanged(ProBuilderMesh mesh) {
```

Comment:
Failed to notice this before. Why do we need to make this public ?
And if so, we would need the API documentation too.